webapp/src/actions/index.ts

```diff
@@ -426,8 +426,11 @@ export function handleConnectFlow(instanceID?: string) {
 
 export function redirectConnect(instanceID: string) {
     return async (dispatch, getState) => {
+        const baseUrl = getState().entities.general.config.SiteURL;
+
         const instancePrefix = '/instance/' + btoa(instanceID);
-        const target = '/plugins/' + PluginId + instancePrefix + '/user/connect';
+        const target = baseUrl + '/plugins/' + PluginId + instancePrefix + '/user/connect';
```

Comment:
It looks like we're still manually computing the plugin URL here. It's concerning that my requests are not fulfilled verbatim, especially on such a small PR. Please refactor this to use `getPluginServerRoute`, so we know that all of the places that need to use the site URL are using it properly. Then please test using a server configured with a subpath.

Reply:
It's updated now. we are now using the `getPluginServerRoute` to generate the root url along with the plugin path.